pybossa/cache/projects.py

```diff
@@ -131,6 +131,7 @@ def format_task(row, lock_users=[]):
         with session.begin_nested():
             session.execute("SET LOCAL enable_indexscan = OFF;")
             results = session.execute(text(sql), params)
+            session.close()
```

Comment:
I think the session is auto closed at the end of "with" context per code [here](https://github.com/zzzeek/sqlalchemy/blob/fc5c54fcd4d868c2a4c7ac19668d72f506fe821e/lib/sqlalchemy/orm/session.py#L1135). You might need session.commit().